packages/server/lib/formatters/connection.ts

```diff
@@ -4,6 +4,7 @@ import type {
     ApiPublicConnection,
     ApiPublicConnectionFull,
     DBConnection,
+    DBConnectionAsJSONRow,
```

Comment:
would that make more sense to modify the queries so they actually return `DBConnection` instead of adding a slightly different type?